students/kyle_odonnell/Lesson2/fizzbuzz.py

```diff
@@ -0,0 +1,28 @@
+# ------------------------------------------------------------------------ #
+# Title: Fizz Buzz Exercise
+# Description: Introduction to Python: Lesson 2 Exercise 2.3
+# ChangeLog:
+# KODonnell,10.17.2020,Created script
+# ------------------------------------------------------------------------- #
+
+
+def FizzBuzz():
```

Comment:
Functions (and almost everything else) should be snake_case in Python.  So, fizz_buzz()